src/middlewares/parameters.ts

```diff
@@ -37,11 +45,25 @@ export const isUserId = (request: UserRequest, response: Response, next: NextFun
     if (decoded.userId.toString() === request.user.id) {
       return next();
     }
+    // Question : Je retourne les bonnes erreurs, là ?
     return unauthorized(response);
   }
   return unauthenticated(response);
 };
 
-// export const isInTeamId = (request: Request, response: Response, next: NextFunction): void => {
-//   return next();
-// };
+// Checks the user is the captain of the team. If not, it will return an error
+export const isInTeamId = async (request: UserRequest, response: Response, next: NextFunction): Promise<void> => {
+  const token = getToken(request);
+  if (token) {
+    const decoded = jwt.verify(token, jwtSecret()) as Token;
+    const { userId } = decoded;
+    const user = await fetchUser(userId.toString());
+    const { teamId } = user;
+
+    if (request.user.teamId === user.teamId) {
```

Comment:
Il faut comparer le `teamId` de l'URL (`request.params.teamId`) avec celui qu'on trouve dans la base de données :
```suggestion
    if (request.params.teamId === user.teamId) {
```